packages/console/src/monaco/MonacoUtils.ts

```diff
@@ -341,89 +344,57 @@ class MonacoUtils {
+    // All editor instances share a global keybinding registry which is where
+    // default keybindings are set. There doesn't appear to be a way to remove
+    // default bindings, but we can add new ones that will override the existing
+    // ones and set `command` to null. This will treat the key events as unhandled
+    // and allow them to bubble up.
+    monaco.editor.addKeybindingRule(
+      // Restart console in Enterprise (see Shortcuts.ts)
       {
-        windows: 'ctrl+D',
-        mac: 'meta+D',
-      },
-      {
-        windows: 'ctrl+H',
-      },
-    ];
+        /* eslint-disable-next-line no-bitwise */
+        keybinding: monaco.KeyMod.CtrlCmd | monaco.KeyCode.KeyD,
+        command: null,
+      }
+    );
 
-    try {
-      keybindings.forEach(keybinding => {
-        if (
-          (MonacoUtils.isMacPlatform() && keybinding.mac === '') ||
-          (!MonacoUtils.isMacPlatform() && keybinding.windows === '')
-        ) {
-          return;
-        }
-        MonacoUtils.removeKeybinding(
-          editor,
-          MonacoUtils.isMacPlatform() ? keybinding.mac : keybinding.windows
-        );
+    // Cmd+H is used to focus Community console history in Windows + Linux.
+    // An alternate shortcut is used for Mac, so no need to override it
+    // (See ConsoleShortcuts.ts)
+    if (!MonacoUtils.isMacPlatform()) {
+      monaco.editor.addKeybindingRule({
+        /* eslint-disable-next-line no-bitwise */
+        keybinding: monaco.KeyMod.CtrlCmd | monaco.KeyCode.KeyH,
+        command: null,
       });
-    } catch (err) {
-      // This is probably only caused by Monaco changing private methods used here
-      log.error(err);
     }
   }
 
   /**
-   * Remove a keybinding and allow the events to bubble up
-   * If you want the keybinding removed for all editors, add it to removeConflictingKeybindings
-   * Monaco still captures the event if you choose to override the keybinding with a no-op function
-   *
-   * Based on the following comment to remove a keybinding and let the event bubble up
-   * https://github.com/microsoft/monaco-editor/issues/287#issuecomment-331447475
-   * The issue for an API for this has apparently been open since 2016. Link below
-   * https://github.com/microsoft/monaco-editor/issues/102
-   * @param editor The editor to remove the keybinding from
-   * @param keybinding The key string to remove. E.g. 'ctrl+C' for copy on Windows
+   * Creates actions with a `noop` run function to disable specified keybindings.
+   * Note that this will swallow the events. To disable default keybindings in a
+   * way that allows events to propagate upward, see `removeConflictingKeybindings`
+   * @param editor Editor to disable keybindings for
+   * @param keybindingsList List of keybinding tuples to disable
    */
-  static removeKeybinding(
+  static disableKeyBindings(
     editor: monaco.editor.IStandaloneCodeEditor,
-    keybinding: string | undefined
+    ...keybindingsList: number[][]
```

Comment:
Minor, but I'm not a big fan of rest args when the type is known like this. It won't matter in this case, but there is a negative impact on memory usage for calls w/ rest args compared to an array arg when there's a large number of arguments. I also find the hover tooltip less clear because 1 level of the array actually belongs to the rest param, so I never actually use a 2D array as a consumer

It's also harder to add another arg in the future if we needed it. Maybe we want to add a `global = false` param. Now everywhere using `disableKeyBindings` must be rewritten to support that

`disableKeyBindings(editor, [[1], [2], [3]])` != `disableKeyBindings(editor, [1], [2], [3])` (although they would do the same in this case since monaco would just register 1 action w/ 3 possible keybindings I believe)

Reply:
@mattrunyon I've changed to a `number[]` arg no longer rest param.

Not sure I follow on the `global = false` param? How would that be consumed?

Reply:
Ah, I think I understand now you were just given an example of adding a new arg, not suggesting I add a global arg?

Reply:
Yep just an example